docs/assumptions.rst

```diff
@@ -28,22 +28,27 @@ Application Behavior
 ---------------------------
 Consistency Model
 ---------------------------
-In the first version of UnifyFS, lamination will be explicitly initiated by a
-UnifyFS API call. In subsequent versions, we will support implicit initiation
-of file lamination. Here, UnifyFS will determine a file to be laminated based
-on conditions, e.g., ``fsync`` or ``ioctl`` calls, or a time out on ``close``
-operations. As part of the UnifyFS project, we will investigate these implicit
-lamination conditions to determine the best way to enable lamination of files
-without explicit UnifyFS API calls being made by the application.
-
-In the first version of UnifyFS, eventually, a process declares the file to be
-laminated through a UnifyFS API call.
-After a file has been laminated, the contents of the file cannot be changed.
-The file becomes permanently read-only.
-After lamination, any process may freely read any part of the file.
-If the application process group fails before a file has been laminated,
```

Comment:
Some of the lines in this paragraph probably still apply, like letting Unify delete a file if a process group fails before laminating a file.